docs/content/tutorial/step_00.ngdoc

```diff
@@ -7,130 +7,143 @@
 
 
 In this step of the tutorial, you will become familiar with the most important source code files of
-the AngularJS phonecat app. You will also learn how to start the development servers bundled with
-angular-seed, and run the application in the browser.
+the AngularJS Phonecat App. You will also learn how to start the development servers bundled with
+[angular-seed][angular-seed], and run the application in the browser.
 
-Before you continue, make sure you have set up your development environment and installed all necessary
-dependencies, as described in {@link index#get-started Get Started}.
+Before you continue, make sure you have set up your development environment and installed all
+necessary dependencies, as described in the {@link tutorial/#environment-setup Environment Setup}
+section.
 
 In the `angular-phonecat` directory, run this command:
 
 ```
 git checkout -f step-0
 ```
 
-
 This resets your workspace to step 0 of the tutorial app.
 
 You must repeat this for every future step in the tutorial and change the number to the number of
 the step you are on. This will cause any changes you made within your working directory to be lost.
 
-If you haven't already done so you need to install the dependencies by running:
+If you haven't already done so, you need to install the dependencies by running:
 
 ```
 npm install
 ```
 
-To see the app running in a browser, open a *separate* terminal/command line tab or window, then
-run `npm start` to start the web server. Now, open a browser window for the app and navigate to
-<a href="http://localhost:8000/app/" target="_blank" title="Open app on localhost">`http://localhost:8000/app/`</a>
+To see the app running in a browser, open a _separate_ terminal/command line tab or window, then run
+`npm start` to start the web server. Now, open a browser window for the app and navigate to
+http://localhost:8000/index.html.
```

Comment:
The version of `http-server` we are using now also supports opening the browser.
Since we are not using HTML5 mode in the tutorial (I would rather keep it that way, in order to avoid discussing one more concept (server-side configuration for HTML5 mode)), I don't think `lite-server` offers any advantage over `http-server`. We _could_ switch, but we need to also update `angular-phonecat` and `angular-seed`, so I would avoid it if there is no compelling reason.

How about we do it in a subsequent PR ?